config_defaults.yml

```diff
@@ -78,6 +79,13 @@ Gemfile:
       - gem: master_manipulator
       - gem: beaker-hostgenerator
         from_env: 'BEAKER_HOSTGENERATOR_VERSION'
+
+MAINTAINERS:
+  maintainers:
+    - github: puppetlabs
+      email: "discuss-modules@puppet.com"
```

Comment:
Actually it looks like it doesn't even allow posting from outside Puppet Inc:

<img width="457" alt="screen shot 2016-09-14 at 9 56 16 am" src="https://cloud.githubusercontent.com/assets/1752967/18521616/a0ec63be-7a61-11e6-8bbc-280757eefc9e.png">

We might want to either make that externally postable (with moderation) or use a different group that already is.